### ISSUE TITLE
Cute classloader does not find the generated java class when executing custom assertion

### ISSUE THREAD

**Issue by verhasi** [toolisticon/cute#83]
Hi Tobias,

Started to use the newly released 1.2.0 version and got the following issue

```
java.lang.AssertionError: guru/mocker/annotation/mixin/test/TestParentClass

-------------------------------
-- UNEXPECTED EXCEPTION: 
-------------------------------
MESSAGE : guru.mocker.annotation.mixin.test.TestParentClass
STACKTRACE : java.lang.ClassNotFoundException: guru.mocker.annotation.mixin.test.TestParentClass
	at java.base/java.lang.ClassLoader.findClass(ClassLoader.java:733)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:593)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1027)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:889)
	at io.toolisticon.cute.CuteClassLoaderImpl.<init>(CuteClassLoaderImpl.java:32)
	at io.toolisticon.cute.CuteApi$CompilationOutcome.<init>(CuteApi.java:1258)
	at io.toolisticon.cute.CuteApi$DoCustomAssertionsImpl.executeCustomAssertions(CuteApi.java:1586)
	at guru.mocker.annotation.mixin.test.BlackBoxTest.mixinProcessor_hasParentNoInterfaceAndNoConstructorParameter_emptyParentGenerated(BlackBoxTest.java:40)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:728)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:156)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:147)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:86)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(InterceptingExecutableInvoker.java:103)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:92)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:86)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:218)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:214)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:139)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:69)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:198)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:169)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:93)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:58)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:141)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:57)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:103)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:85)
	at org.junit.platform.launcher.core.DelegatingLauncher.execute(DelegatingLauncher.java:47)
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:63)
	at com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:57)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:232)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:55)


-------------------------------
-- WARNING MESSAGES: 
-------------------------------
[1]'warning: Supported source version 'RELEASE_8' from annotation processor 'io.toolisticon.cute.AnnotationProcessorWrapper' less than -source '21''

-------------------------------
-- GENERATED FILEOBJECTS: 
-------------------------------
{
  'GENERATED JAVA FILE OBJECTS' : [
    'mem://SOURCE_OUTPUT/guru/mocker/annotation/mixin/test/TestParentClass.java' := '/Users/istvanverhas/IdeaProjects/Mixin/target/cute_failingUnitTests/biikpjptxg/guru/mocker/annotation/mixin/test/TestParentClass.java', 
    'mem://CLASS_OUTPUT/guru/mocker/annotation/mixin/test/TestClass.class' := '/Users/istvanverhas/IdeaProjects/Mixin/target/cute_failingUnitTests/biikpjptxg/guru/mocker/annotation/mixin/test/TestClass.class', 
    'mem://CLASS_OUTPUT/guru/mocker/annotation/mixin/test/TestParentClass.class' := '/Users/istvanverhas/IdeaProjects/Mixin/target/cute_failingUnitTests/biikpjptxg/guru/mocker/annotation/mixin/test/TestParentClass.class', 
  ],
  'GENERATED FILE OBJECTS' :'No files were generated !!!'
}
-------------------------------
-- COMPILE TEST CONFIGURATION: 
-------------------------------
{\ntestType = BLACK_BOX,
compilationSucceeded = true,
processors = [class guru.mocker.annotation.mixin.MixinProcessor],
compilerOptions = [],
getPassInType = null,
fileObjectChecks = [],
modules = [],
getGeneratedClassesTest = [],
getExceptionIsThrown = null,
sourceFiles = [io.toolisticon.cute.JavaFileObjectUtils$JavaSourceFromResource[resource:///guru/mocker/annotation/mixin/test/mixin/hasParentNoInterfaceNoConstructorParameter.java]],
javaFileObjectChecks = [],
passInConfiguration = null,
unitTest = null,
compilerMessageChecks = [],
}

	at io.toolisticon.cute.extension.plainjava.AssertionErrorAssertion.fail(AssertionErrorAssertion.java:11)
	at io.toolisticon.cute.CuteApi$DoCustomAssertionsImpl.executeCustomAssertions(CuteApi.java:1590)
	at guru.mocker.annotation.mixin.test.BlackBoxTest.mixinProcessor_hasParentNoInterfaceAndNoConstructorParameter_emptyParentGenerated(BlackBoxTest.java:40)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)

```java.lang.AssertionError: guru/mocker/annotation/mixin/test/TestParentClass

-------------------------------
-- UNEXPECTED EXCEPTION: 
-------------------------------
MESSAGE : guru.mocker.annotation.mixin.test.TestParentClass
STACKTRACE : java.lang.ClassNotFoundException: guru.mocker.annotation.mixin.test.TestParentClass
	at java.base/java.lang.ClassLoader.findClass(ClassLoader.java:733)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:593)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1027)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:889)
	at io.toolisticon.cute.CuteClassLoaderImpl.<init>(CuteClassLoaderImpl.java:32)
	at io.toolisticon.cute.CuteApi$CompilationOutcome.<init>(CuteApi.java:1258)
	at io.toolisticon.cute.CuteApi$DoCustomAssertionsImpl.executeCustomAssertions(CuteApi.java:1586)
	at guru.mocker.annotation.mixin.test.BlackBoxTest.mixinProcessor_hasParentNoInterfaceAndNoConstructorParameter_emptyParentGenerated(BlackBoxTest.java:40)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:728)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:156)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:147)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:86)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(InterceptingExecutableInvoker.java:103)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:92)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:86)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:218)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:214)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:139)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:69)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:198)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:169)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:93)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:58)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:141)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:57)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:103)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:85)
	at org.junit.platform.launcher.core.DelegatingLauncher.execute(DelegatingLauncher.java:47)
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:63)
	at com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:57)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:232)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:55)


-------------------------------
-- WARNING MESSAGES: 
-------------------------------
[1]'warning: Supported source version 'RELEASE_8' from annotation processor 'io.toolisticon.cute.AnnotationProcessorWrapper' less than -source '21''

-------------------------------
-- GENERATED FILEOBJECTS: 
-------------------------------
{
  'GENERATED JAVA FILE OBJECTS' : [
    'mem://SOURCE_OUTPUT/guru/mocker/annotation/mixin/test/TestParentClass.java' := '/Users/istvanverhas/IdeaProjects/Mixin/target/cute_failingUnitTests/biikpjptxg/guru/mocker/annotation/mixin/test/TestParentClass.java', 
    'mem://CLASS_OUTPUT/guru/mocker/annotation/mixin/test/TestClass.class' := '/Users/istvanverhas/IdeaProjects/Mixin/target/cute_failingUnitTests/biikpjptxg/guru/mocker/annotation/mixin/test/TestClass.class', 
    'mem://CLASS_OUTPUT/guru/mocker/annotation/mixin/test/TestParentClass.class' := '/Users/istvanverhas/IdeaProjects/Mixin/target/cute_failingUnitTests/biikpjptxg/guru/mocker/annotation/mixin/test/TestParentClass.class', 
  ],
  'GENERATED FILE OBJECTS' :'No files were generated !!!'
}
-------------------------------
-- COMPILE TEST CONFIGURATION: 
-------------------------------
{\ntestType = BLACK_BOX,
compilationSucceeded = true,
processors = [class guru.mocker.annotation.mixin.MixinProcessor],
compilerOptions = [],
getPassInType = null,
fileObjectChecks = [],
modules = [],
getGeneratedClassesTest = [],
getExceptionIsThrown = null,
sourceFiles = [io.toolisticon.cute.JavaFileObjectUtils$JavaSourceFromResource[resource:///guru/mocker/annotation/mixin/test/mixin/hasParentNoInterfaceNoConstructorParameter.java]],
javaFileObjectChecks = [],
passInConfiguration = null,
unitTest = null,
compilerMessageChecks = [],
}

	at io.toolisticon.cute.extension.plainjava.AssertionErrorAssertion.fail(AssertionErrorAssertion.java:11)
	at io.toolisticon.cute.CuteApi$DoCustomAssertionsImpl.executeCustomAssertions(CuteApi.java:1590)
	at guru.mocker.annotation.mixin.test.BlackBoxTest.mixinProcessor_hasParentNoInterfaceAndNoConstructorParameter_emptyParentGenerated(BlackBoxTest.java:40)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
```
If you need more details please let me know.

**Comment by tobiasstamann** [comment]
Hi, 

at the moment i can just guess that it is related with Java 21. 

Currently,I do test builds with all java versions up to 17.

Do you have some kind of feature branch i can use for testing?

Best regards

Tobias

**Comment by tobiasstamann** [comment]
One additional question. Do you get this error on both command line and IDE builds/test executions?

**Comment by verhasi** [comment]
Hi @tobiasstamann 

It failed both in IDE and command line maven as well. I tried it with Java version 1.8 and the result was the same. 
Your fix on the dev branch fixed it perfectly. :) It was super fast again. Do you sleep at all? ;)

Best regards, 
Istvan

**Comment by tobiasstamann** [comment]
Hi,

good to hear that the issue is fixed now.

The feature is relatively new and i had used it myself just in simple tests.

The release is triggered now and will be available with version 1.3.0.

Thanks for filing that issue :)

Best regards

Tobias
